e2eTests/snap/snapConfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "gridUrl": "http://hub:4444/wd/hub",
+  "gridUrl": "http://selenium-grid.tnl-dev.ntch.co.uk:4444/wd/hub",
```

Comment:
we have a doker image that spins up the grid which is why it was using ` "gridUrl": "http://hub:4444/wd/hub"`